user-guide/Standard_Apps/SRM/Booking_Manager/Booking_Manager_user_interface/Booking_Manager_Config_tab.md

```diff
@@ -0,0 +1,271 @@
+---
+uid: Booking_Manager_Config_tab
+---
+
+# The Config tab
+
+This tab of the Booking Manager app provides an overview of all configuration parameters for the Booking Manager.
+
+## General subtab
+
+### Application Setup settings
+
+- **Default Virtual Platform**: The default Virtual Platform of the Booking Manager. This is used among others to filter service definitions and resources. In the Booking Wizard, service definitions will only be shown if their *Virtual Platform* property is set to the identifier specified in this setting. For resources, the name of a resource pool that should be used with this Virtual Platform should start with the specified identifier followed by a period. For example, if *Default Virtual Platform* is set to *VPA*, a valid name for a resource pool would be *VPA.Decoder*.
+
+- **Resource Pool**: Defines the Source Resource Pool name, so that Source resources can be identified and displayed on the first page of the Booking Wizard.
```

Comment:
Because some users may still be using an older version of SRM, I'd indicate that this setting is deprecated instead. This way the help remains applicable for everyone.